doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -180,6 +180,21 @@ Event with non-extended logging:
       "http_content_type": "application\/x-gzip"
   }
 
+In case the hostname shows a port number (such as in case the header "Hostname: www.digip.org:1337") is present:
```

Comment:
similiar to your other PR I would use more neutral hostnames as examples.

Reply:
done